src/06-loops.js

```diff
@@ -61,7 +82,15 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+function containsWhileLoop(array, target) {
+
+  while (array.target = true) {
```

Comment:
what are we doing with line 87 and 90 ?  What does  ` array.target = true ` evaluate to?